benchmarks/models.py

```diff
@@ -203,6 +203,21 @@ def testjobs_updated(self):
         jobs = self.test_jobs.filter(updated_at__gt=self.updated_at)
         return jobs.exists()
 
+    @property
+    def testrunnerclass(self):
+        runners = sorted(
+            filter(
+                lambda t: t not in ['GenericLavaTestSystem', 'TestSystem'],
+                set([ t.testrunnerclass for t in self.test_jobs.all() ])
+            )
+        )
+        if len(runners) == 1:
+            return runners[0]
+        elif len(runners) > 1:
+            raise LookupError("Result has multiple testrunner classes: %r'" % runners)
```

Comment:
right. in that case how to we select which method to use when comparing results across builds? Do we even want to do that?